### PR TITLE
EPMRPP-90667 || No 'Email server is disabled' message is displayed for Member/Operator/Customer users

### PR DESCRIPTION
**Pull request by maria-hambardzumian**
…for Member/Operator/Cusomer users

## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [x] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

File: app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx [hunk 1]
```diff
@@ -95,6 +95,11 @@ export const RuleGroup = ({ pluginName, typedRules, notifications, isPluginEnabl
 
   const isUpdateSettingAvailable = canUpdateSettings(userRole, projectRole);
   const isReadOnly = !isUpdateSettingAvailable || !isPluginEnabled;
+  const isActivationRequired =
```
Comment (Vadim73i) [inline app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx:98]: can be simplified `const isActivationRequired = isUpdateSettingAvailable || typedRules?.length > 0;`

File: app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx [hunk 2]
```diff
@@ -95,6 +95,11 @@ export const RuleGroup = ({ pluginName, typedRules, notifications, isPluginEnabl
...
+    isUpdateSettingAvailable || (!isUpdateSettingAvailable && typedRules?.length > 0);
+  const isDisabledTooltipActivationRequired = !pluginName && isActivationRequired;
```
Comment (Vadim73i) [inline app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx:100]: pluginName is required

File: app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx [hunk 3]
```diff
@@ -246,25 +251,24 @@ export const RuleGroup = ({ pluginName, typedRules, notifications, isPluginEnabl
...
-                >
-                  <i className={cx('icon', 'about-icon')}>{Parser(AboutIcon)}</i>
-                </RuleItemDisabledTooltip>
-              </div>
-            ) : (
-              pluginName === EMAIL &&
-              !isEmailIntegrationAvailable && (
-                <div className={cx('integrate-configurations')}>
-                  <p>{formatMessage(messages.notConfiguredIntegration)}</p>
+          {isDisabledTooltipActivationRequired ? (
```
Comment (Vadim73i) [inline app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx:254]: Can be used the `isActivationRequired`?

File: app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx (hunk 2 above)
Comment (maria-hambardzumian) [reply app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx:100]: yes, should be isPluginEnabled

File: app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx (hunk 3 above)
Comment (maria-hambardzumian) [reply app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx:254]: In this case, there is a need to include 'isPluginEnabled' in the condition.

**Review by Vadim73i — APPROVED** [review]

**Pull request merged**